### ISSUE TITLE
Milestone 3 Roadmap

### ISSUE THREAD

**Issue by YCPjsteck** [YCPjsteck/CS320-Sp19-Labyrinth-of-Tramateck#31]
Issues are broken down into priorities, and sorted within priorities by when they should be done.

High priority:
- [x] #10: REQUIRED by Hake for this milestone. Ref: #9; the fake DB isn't necessary. Even if not complete, we should at least have it working.
- #23: We really should've had this working by MS1, but MS2 is where we had it planned. Given that this is still not done and we're approaching the time between MS3 and the final is 2 weeks and change, we need to have this be one of the first things done.
- [x] #30: Room is one of the most important classes given that it is the smallest unit of movement and where all player interactions occur. Make sure that it is 100% done.

Medium priority:
- [x] #33: TEST EVERYTHING!
- [x] #27: The last class that needs created. Should be done in conjunction with #23 to have a login page.
- [x] #32: With the events class created, it now needs to be able to be read into the game.
- #22: Once #30 is done, this can be done through a controller. 
- [x] #15: Needs to be done so that we can have stuff for the player to actually collect and use.
- [x] #24: Updating the UML is always important.

Low priority:
- [x] #16: Should we have such a superclass?
- [x] #29: Testing purposes; no need to have duplicate code.
- [x] #28: Organization related. 
- [x] #26: Graphic design related.
- [x] #21: Really just an ease of use/access thing.
- [x] #17: Just make sure stuff is commented for documentation purposes.
- [x] #34: Removes complexity.

**Comment by YCPjsteck** [comment]
Before spring break, have a good portion of the low and mid priority issues done on our own branches. Then merge our branches into the MS3 branch at the beginning of break. From there, work together on getting the SQL database started on MS3. Then branch off of MS3 again to individually finish the rest of the issues. Finally, merge it all back into MS3 before the presentation and merge MS3 into master.

**Comment by YCPjsteck** [comment]
For Saturday:
* Jonathan
  * Create item subclasses and work on item-player interaction
* Chase
  * Finish enumerations, collapse Enemy and Vendor into NPC and create a Character superclass to Player and NPC
* Jaden
  * Have Events class finalized and working with Library class
